lib/node/runner/isaac_voting_empty_transaction_test.go

```diff
@@ -0,0 +1,102 @@
+package runner
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"boscoin.io/sebak/lib/ballot"
+	"boscoin.io/sebak/lib/consensus"
+	"boscoin.io/sebak/lib/consensus/round"
+)
+
+// Test that ballot with empty transactions have VotingYES
+func TestISAACBallotWithEmptyTransaction(t *testing.T) {
+	conf := consensus.NewISAACConfiguration()
+	nr, _, _ := createNodeRunnerForTesting(1, conf, nil)
+
+	latestBlock := nr.Consensus().LatestConfirmedBlock()
+	round := round.Round{
+		Number:      0,
+		BlockHeight: latestBlock.Height,
+		BlockHash:   latestBlock.Hash,
+		TotalTxs:    latestBlock.TotalTxs,
+	}
+
+	b := ballot.NewBallot(nr.localNode, round, []string{})
+	require.Equal(t, b.B.Vote, ballot.VotingYES)
+}
+
+// Test that the voting process ends normally with a ballot with an empty transaction.
+func TestISAACBallotWithEmptyTransactionVoting(t *testing.T) {
+	conf := consensus.NewISAACConfiguration()
+	nr, nodes, _ := createNodeRunnerForTesting(5, conf, nil)
+
+	// `nodeRunner` is proposer's runner
+	proposer := nr.localNode
+
+	nr.Consensus().SetLatestConsensusedBlock(genesisBlock)
+	latestBlock := nr.Consensus().LatestConfirmedBlock()
+	require.Equal(t, uint64(1), latestBlock.Height)
+	require.Equal(t, uint64(0), latestBlock.TotalTxs)
+
+	// Generate proposed ballot in nr
+	var err error
+
+	err = nr.proposeNewBallot(0)
```

Comment:
`err` should be `_` or check it `require.Nil`.

Reply:
Also you can just use `:=` and remove two useless lines